munit/shared/src/main/scala/munit/internal/console/Lines.scala

```diff
@@ -19,11 +19,17 @@ class Lines extends Serializable {
 
   def findPath(cwd: String, path: String, max: Int): Path = {
     val p = Paths.get(cwd).resolve(path)
+    def getParentPath(somePath: String, sep: String): String = {
+      val somePath1 =
+        if (somePath.endsWith(sep)) somePath.dropRight(sep.length)
+        else somePath
+      somePath1.split(sep).dropRight(1).mkString(sep)
+    }
     if (Files.exists(p)) p
     else if (max < 1) sys.error(s"$path was not found")
     else if (cwd.contains("\\"))
-      findPath(cwd.split("\\").dropRight(1).mkString("\\"), path, max - 1)
-    else findPath(cwd.split("/").dropRight(1).mkString("/"), path, max - 1)
+      findPath(getParentPath(cwd, "\\"), path, max - 1)
```

Comment:
What do you think about using https://docs.oracle.com/javase/8/docs/api/java/io/File.html#separator instead of looking at the current path ? Or would this only work on the JVM ?

Reply:
Yea, for native and JS it might be simpler to just look for `\`.